models/classes/Lis/LisAuthAdapter.php

```diff
@@ -56,6 +57,12 @@ public function authenticate()
                 $exception->getCode(),
                 $exception
             );
+        } catch (LockOutException $exception) {
```

Comment:
> In PHP 7.1 and later, a catch block may specify multiple exceptions using the pipe (|) character. This is useful for when different exceptions from different class hierarchies are handled the same.

https://www.php.net/manual/en/language.exceptions.php#language.exceptions.catch

Could be useful here IMO.